docs/stateless.md

```diff
@@ -0,0 +1,57 @@
+
+### Use cases
+* Scaling the IRMA server horizontally, i.e. running several IRMA severs in parallel.
+* Preventing data loss when restarting an IRMA server.
+* Preventing data loss in case of IRMA server crashes.
+
+## Running the IRMA server in stateless mode
+### Example
+You can start the IRMA server in stateless mode by setting the `store-type` option to `redis`. Additionally you need to provide a Redis server address and password. For test purposes you can override the need for a password by setting the `redis-allow-empty-password` option to `true`. However, make sure to use a secure Redis password in production. Your Redis data store will contain sensitive data and must be password-protected.
+
+```
+irma server -vv --store-type redis --redis-addr "localhost:6379" --redis-pw "placeholderPassword"
+```
+
+> By default TLS is _not_ enabled when using Redis. Since the session data is not encrypted before being sent to Redis, we recommend turning on TLS for your Redis server. 
+
+Your Redis server configuration may look as follows:
+```
+requirepass placeholderPassword
+
+// Disable the non-TLS port completely
+port 0
+// Enable TLS on the default Redis port
+tls-port 6379
+
+// X.509 certificate and a private key
+tls-cert-file /path/to/redis.crt
+tls-key-file /path/to/redis.key
+
+// Disable TLS client authentication
+tls-auth-clients no
+```
+
+The IRMA server already authenticates to the Redis server using a password. The IRMA server currently does not support mutual TLS for the Redis connection. Therefore, the `tls-auth-clients` in the Redis configuration is set to `no`. For more information on TLS support in Redis see the [Redis documentation](https://redis.io/topics/encryption).
+
+If you want to run several IRMA servers, you can now run them behind a load balancer and connect them to the same Redis instance.
+
+> Currently a simple locking mechanism is implemented. When running several Redis instances in parallel, edge cases may occur. We recommend to use a single Redis instance.
+
+### Using multiple Redis instances
+Currently the IRMA server does not support Redis clusters or a master/slave mode. You can only connect to one Redis address. This means that the Redis connection is currently a bottleneck and single point of failure. You could use [Redis Enterprise](https://redis.com/redis-enterprise-cloud/overview/) which will let you connect with one outward-facing Redis connection and will provide you with an underlying failover mechanism.  
+
+Please get in touch with us, should you wish the IRMA server to support multiple Redis instances.
+
+### Using stateless mode in combination with server-sent events
```

Comment:
This title is rather long, which looks a little awkward in the navigation on the right. I think just calling this "Server-sent events" should suffice; that it has something to do with stateless mode will be obvious.

<img width="287" alt="Screenshot 2021-12-07 at 16 57 12" src="https://user-images.githubusercontent.com/979897/145052550-3e88a04d-a2f4-41bc-b20d-23ba4e4f1ecf.png">